http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -1248,6 +1249,12 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         if (req.appType == AppType.Allowed)
           Some(config.leoKubernetesConfig.allowedAppConfig.numOfReplicas)
         else None
+      autoDeleteThresholdInMinutes = req.autoDeleteThresholdDuration match {
+        case None               => autoDeleteOffValue
+        case Some(Duration.Inf) => autoDeleteOffValue
```

Comment:
so the type for `autoDeleteThresholdDuration` is a `scala.concurrent.duration.Duration`, which can be either `Inf` or a `FiniteDuration`...

`on` isn't a valid value for `scala.concurrent.duration.Duration`...If you're suggesting adding another flag for indicating `on`/`off`, my thinking is that complicates logic becuz u'd be translating a combo of (`Boolean`, `FiniteDuration`) into the DB representation instead of just one value of `Duration`

Reply:
I did consider to add extra AutoDeleteEnabled parameter in both Db and API interface but decide not to do it because the extra value is a bit unnecessary and add extra complexity actually. And even for AutoPause, I can pass empty AuthPauseThreshold, 0, to represent no auto pause. This is more complicated and confusing to us because there are too many ways to represent no auto pause. 

One threshold value field is good enough in this case. 
Also  `Inf` is the String represent in Duration: 
https://www.scala-lang.org/api/2.12.4/scala/concurrent/duration/Duration$.html#Inf:scala.concurrent.duration.Duration.Infinite
so I do not think it is confusing. 

happy to chat more. Or in this doc: https://docs.google.com/document/d/1eWfY8PQoDKVOdIeREuEC9ObLhXlsTtG2ugLwuqov81k/edit#heading=h.kmuxa0wy0za9

Reply:
I think my preference is to not support `INF` at all.
I believe (if I am reading this correctly) that this is in a public facing API and would prefer to simple not support it as a valid value.